authhelper.go

```diff
@@ -0,0 +1,179 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/Pallinder/go-randomdata"
+	"github.com/golang-jwt/jwt/v4"
+	"github.com/muir/nfigure"
+	"github.com/pkg/browser"
+)
+
+type Output struct {
+	ExpiresAt          string `json:"expire"` // RFC3339 format
+	Password           string `json:"password"`
```

Comment:
Based on how I'm seeing this used, 'Password' feels like a misnomer and its use could give some people pause/trigger security warnings. Does 'Token' make more sense or am I misreading this?

Reply:
Changed to `PasswordToken` and added a small comment.  Good enough?